versioned_docs/version-2/1-click-signup/api-reference/types.mdx

```diff
@@ -237,7 +237,11 @@ See also our [Postman docs](https://api.docs.verified.inc/).
         [identifierKey: string]: string
     },
     "credentials": {
-        [credentialKey: string]: string | [string] | Object | [Object]
+        [credentialKey: string]: 
+            string     // Single credential value when `children` is not present and `multi` is false
```

Comment:
I didn't phrase it like "set to `true/false`" here because I wanted to shorten the comments.